### DISCUSSION TITLE
Lua functions show weird behavior since 0.4.0

### DISCUSSION THREAD

**Discussion by spacian** [vscode-neovim/vscode-neovim#1360] [Q&A]
I gathered some functions to make my life easier when trying to interact with vscode via lua. However, I think with version 0.4.0, they aren't working anymore and show weird behavior.

First, my helper function is the following;
```
function nvim_feedkeys(keys)
    local feedable_keys = vim.api.nvim_replace_termcodes(keys, true, false, true)
    vim.api.nvim_feedkeys(feedable_keys, "n", false)
end
```

Then I also have a center function, which centers the buffer around my cursor position:

```
function center()
    nvim_feedkeys(':call <snr>3_reveal("center",0)<enter>') 
end
```
It looks like that because `zz` is mapped to `<cmd>call <snr>3_reveal('center',0)<cr>`. You can check yourself in vscode-neovim by typing `:nn zz`.

Now I simply combined the two, e.g.:
```
vim.keymap.set({'n','v'}, '<c-u>',
    function()
        nvim_feedkeys('20k^')
        center()
    end
)
```
This worked perfectly until 0.4.0. Now it doesn't. However, centering still works if called alone like in this function:
```
vim.keymap.set({'n','v'}, '<leader><space>',
    function()
        center()
    end
)
```

It also works without a leading number in the previous command like so:
```
vim.keymap.set({'n','v'}, '<c-u>',
    function()
        nvim_feedkeys('k^')
        center()
    end
)
```

Any suggestions on how to fix this setup?
Thank you!

**Comment by theol0403** [comment]
I don't know if any code has changed which would affect that, can you try a commit bisect of the extension?

**Reply by spacian** [reply]
I'll try to have a look. Learning new things every day... :D
